src/Aspire.Cli/Commands/ConfigCommand.cs

```diff
@@ -28,17 +29,19 @@ public ConfigCommand(IConfiguration configuration, IConfigurationService configu
         _configurationService = configurationService;
         _interactionService = interactionService;
 
-        var getCommand = new GetCommand(configurationService, _interactionService);
-        var setCommand = new SetCommand(configurationService, _interactionService);
-        var listCommand = new ListCommand(configurationService, _interactionService);
-        var deleteCommand = new DeleteCommand(configurationService, _interactionService);
+        var getCommand = new GetCommand(configurationService, _interactionService, features, updateNotifier);
+        var setCommand = new SetCommand(configurationService, _interactionService, features, updateNotifier);
+        var listCommand = new ListCommand(configurationService, _interactionService, features, updateNotifier);
+        var deleteCommand = new DeleteCommand(configurationService, _interactionService, features, updateNotifier);
 
         Subcommands.Add(getCommand);
         Subcommands.Add(setCommand);
         Subcommands.Add(listCommand);
         Subcommands.Add(deleteCommand);
     }
 
+    protected override bool UpdateNotificationsEnabled => throw new NotImplementedException();
```

Comment:
Shouldn't this be false?
```suggestion
    protected override bool UpdateNotificationsEnabled => false;
```